src/content/apis/translations/yoda-translations.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Yoda translator API
```

Comment:
The root API seems to provide translation for multiple categories, so instead of naming it as Yoda Translator API, we should rather name it as `Fun Translations API`

Reply:
done